### ISSUE TITLE
Accessibility improvements

### ISSUE THREAD

**Issue by swissspidy** [pluginkollektiv/cachify#36]

**Comment by Zodiac1978** [comment]
Can you elaborate that @swissspidy?

**Comment by swissspidy** [comment]
The settings screen and the toolbar menu item aren't very a11y-friendly, at least last time I checked. Think keyboard navigation, screen reader text, etc.

WordPress has some high standards there and we should try to follow best practices. See https://make.wordpress.org/accessibility/handbook/

**Comment by 00travelgirl00** [comment]
Keyboard Navigation worked quite good now. I have changed some labels, so the screen reader accessible should be fine now as well. #147 

But I'm not quite sure if we should add screen reader text to increase the accessibility.

**Comment by 00travelgirl00** [comment]
@Zodiac1978 can we close this ticket with PR #147 ?
